src/build123d/geometry.py

```diff
@@ -450,7 +450,7 @@ def __eq__(self, other: object) -> bool:
 
     def __hash__(self) -> int:
         """Hash of Vector"""
-        return hash(self.X) + hash(self.Y) + hash(self.Z)
+        return hash(round(self.X, 6)) + hash(round(self.Y, 6)) + hash(round(self.Z, 6))
```

Comment:
```suggestion
        return hash((round(self.X, 6), round(self.Y, 6), round(self.Z, 6)))
```
any reason not to use the built-in tuple hash instead of adding individual number hashes?

Reply:
I've made this change.  Thanks for pointing out the algorithm was flawed.